### ISSUE TITLE
Synbio hub searches are timing out on production server

### ISSUE THREAD

**Issue by mwes** [SD2E/experimental-intent-parser#150]
In GitLab by @nwalczak on Oct 31, 2019, 11:55

If analyze or add to synbiohub are used then the SBH queries never resolve and instead the "Generating List..." message stays up forever.

If I run the web console in Firefox, I eventually see this message:

`Error: Timeout: http://intentparser.sd2e.org/searchSynBioHub`

This means that the `URLFetch` timeout is being hit before the SBH query returns.  Interestingly, if I run the query in my web browser using the SPARQl endpoint, it returns in a reasonable amount of time.

There are really a couple of issues here.  One is that the query is run with a success handler, but no error handler (see `intent-parser-server/src/analyze_sidebar.html` line 63 and 64).  If we add an error handler we can have some slightly better behavior in reporting there's a problem.

When I test this in my test environment I don't have any issues.  I'm not sure why SBH is responding to the TACC server more slowly...

**Comment by mwes** [comment]
In GitLab by @jakebeal on Oct 31, 2019, 11:59

assigned to @nwalczak

**Comment by mwes** [comment]
In GitLab by @jakebeal on Oct 31, 2019, 12:00

Can you please engage TACC and get their help to sort this out?  This is a critical failure.

**Comment by mwes** [comment]
In GitLab by @nwalczak on Oct 31, 2019, 16:33

Nothing definitive yet, but things are pointing towards this being an issue with something on TACC's side.  If I run the exact some Docker container on my laptop, I don't encounter these issues.

**Comment by mwes** [comment]
In GitLab by @jakebeal on Oct 31, 2019, 22:54

Please get TACC involved ASAP, then. This is a critical time for adoption, and I don't want TACC's infrastructure problems to cause the CPs to start working around intent parser.

**Comment by mwes** [comment]
In GitLab by @nwalczak on Nov 4, 2019, 11:51

I saw a message on Slack about how, over the weekend, the production server seemed to crash.  I checked on it and I saw that inbound connections were being made, but no responses were ever going out: if I made an analyze request I would see the connection, but it never finished scanning the document.

I restarted the container and it seems to have resolved that issue, plus the one reported here.
